src/components/blocks/hero.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+  import { afterNavigate } from '$app/navigation';
+  import SmallHighlights from '$components/pages/home/small-highlights.svelte';
+  import Reel from '$components/reel.svelte';
+
+  import Seo from '$components/seo.svelte';
+  import Slogan from '$components/slogan.svelte';
+  import { drawerLinks } from '$lib/actions/drawer-links';
+  import { VIDEO_EXTENSIONS } from '$lib/constants';
+  import { TrackingEvent } from '$lib/track';
+  import { getImageAttributes } from '$lib/utils/cms';
+  import { getFileExtension } from '$lib/utils/strings';
+
+  import type { HeroStoryblok } from '$types/bloks';
+  import clsx from 'clsx';
+
+  export let block: HeroStoryblok;
+
+  let show = false;
+
+  afterNavigate(() => {
+    if (window.scrollY > 500) {
+      show = true;
+    }
+  });
+</script>
+
+<Seo />
```

Comment:
`Seo` shouldn't be apart of a block, only page

Reply:
true, good catch